internal/identifier/parseable.go

```diff
@@ -129,7 +130,7 @@ func inspect(p Parseable, ids ...string) (string, error) {
 		info, ok := p.Infos()[id]
 		if ok {
 			if config.GetWikidataNamespace() != "" {
-				lines = append(lines, strings.ToUpper(info.String()+"QID: ("+id+")"))
+				lines = append(lines, fmt.Sprintf("%sQID: (%s)", info.String(), id))
```

Comment:
More idiomatic? The function potentially changes the behavior. `ToUpper` needs to be added back in here.

Reply:
The `strings.ToUpper()` here would affect the output and consistency of the revision history we want to return. It doesn't look like it is needed, so keeping as-is.